### ISSUE TITLE
epmin & epmax

### ISSUE THREAD

**Issue by mateusz-olczyk** [OpenMobileAlliance/OMA_LwM2M_for_Developers#469]
I want to ask about new resource attributes introduced in LwM2M 1.1 (_OMA-TS-LightweightM2M_Core-V1_1_1-20190617-A_, section 5.1.2. Attributes Classification): **Minimum Evaluation Period** and **Maximum Evaluation Period**. 

I cannot understand this brief description:

> The Minimum Evaluation Period Attribute indicates the minimum time in seconds the LwM2M Client MUST wait between two evaluations of reporting criteria.

What are "evaluations of reporting criteria"? Do you mean allowed frequency of low-level reads (eg. direct reads from some sensor) performed on a particular resource to check **lt/gt/st** criteria? I would be very happy if you put in the specification some example presenting an application of the new attributes.

**Comment by asoloway64** [comment]
Yes, this resource defines the minimum "allowed frequency of low-level reads".
